website/docs/components/inline-input/examples/number-only_input.tsx

```diff
@@ -6,7 +6,7 @@ const Example = () => {
     <div>
       <InlineInput>
```

Comment:
```suggestion
      <InlineInput w={200}>
```
I think it would be nice to limit its width